### PR TITLE
Cu 864dtv11y poc to einer iguoliveira

### PR DESCRIPTION
**Pull request by gustavobaltazar**
##### Description of changes

- [x] Why (Why did you make this pull request): to implement the poc on einer
- [x] Brief description of the changes: Test added and backend refactor & front end 
- [x] Possible breakouts (Possible places where things could break):Already tested, but possible with data send

### CODE REVIEW

File: apps/server-planner/prisma/schema.prisma [hunk 1]
```diff
@@ -11,9 +11,30 @@ datasource db {
...
+
+  Projects Projects? @relation(fields: [projectsId], references: [id])
+  listCard Card[]
+}
+
+model Projects {
+  id         String @id @default(cuid())
+  title      String
+  background String
+
```
Comment (iguoliveira) [inline apps/server-planner/prisma/schema.prisma:38]: ?

File: apps/server-planner/src/routers/card.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          data: {
+            title: input.title,
+          },
+        });
+        return { createdCard };
+      } catch (error) {
+        throw new TRPCError({
+          code: "INTERNAL_SERVER_ERROR",
+          cause: error,
+          message: "Cannot to create card!",
```
Comment (iguoliveira) [inline apps/server-planner/src/routers/card.ts:42]: to

File: apps/server-planner/src/routers/card.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      z.object({
+        title: z.string(),
+        cardId: z.string(),
+      })
+    )
+    .mutation(async ({ input }) => {
+      try {
+        const createdCard = await prisma.card.create({
+          data: {
+            title: input.title,
```
Comment (iguoliveira) [inline apps/server-planner/src/routers/card.ts:34]: need to contain all fields, if the user want to create a complete card

**Review by iguoliveira — CHANGES_REQUESTED** [review]
need to fix somethings

File: apps/server-planner/src/routers/card.ts (hunk 3 above)
Comment (gustavobaltazar) [reply apps/server-planner/src/routers/card.ts:36]: Fixed 👍

**Review by vinicius-prates — APPROVED** [review]
Nice one!

**Review by iguoliveira — APPROVED** [review]
🚀

**Pull request merged**